Can't be compiled for 32-bit ARM

https://github.com/opencontainers/runc/pull/2145#issuecomment-543161459

```
CGO_ENABLED=1 GOARCH=arm GOARM=6 CC=arm-linux-gnueabi-gcc go build -buildmode=pie  -ldflags "-X main.gitCommit="bf9519326d3dcc4a78f3cddbc54ac7a78a0aa948" -X main.version=1.0.0-rc9+dev " -tags "seccomp apparmor selinux ambient" -o runc-armel .
# github.com/opencontainers/runc/vendor/github.com/cilium/ebpf
vendor/github.com/cilium/ebpf/syscalls.go:285:17: constant 3405662737 overflows int32
Makefile:125: recipe for target 'localcross' failed
```

This line does not compile for `GOARCH=arm GOARM=6`
https://github.com/cilium/ebpf/blob/60d34524c5b34180514b1aae150a53bc41026d81/syscalls.go#L285

--- a/.semaphore/semaphore.yml
+++ b/.semaphore/semaphore.yml
@@ -19,9 +19,11 @@ blocks:
           - sudo apt-get -y update
           - sudo apt-get install -y qemu-system-x86
       jobs:
-      - name: Test build on non-Linux
+      - name: Test building on other OS and arch
         commands:
           - GOOS=darwin go build ./...
+          - GOARCH=arm GOARM=6 go build ./...
+          - GOARCH=arm64 go build ./...
       - name: Test on 5.0.13
         commands:
           - timeout -s KILL 60s ./run-tests.sh 5.0.13

--- a/syscalls.go
+++ b/syscalls.go
@@ -282,7 +282,7 @@ func bpfPinObject(fileName string, fd *bpfFD) error {
 	if err := unix.Statfs(dirName, &statfs); err != nil {
 		return err
 	}
-	if statfs.Type != bpfFSType {
+	if uint64(statfs.Type) != bpfFSType {
 		return errors.Errorf("%s is not on a bpf filesystem", fileName)
 	}
 

